packages/compass-aggregations/src/modules/index.ts

```diff
@@ -292,8 +291,9 @@ const doNamespaceChanged = (state: RootState, action: AnyAction) => {
  *
  * @returns {Object} The new state.
  */
-const doReset = (): RootState => ({
-  ...INITIAL_STATE
+const doReset= (state: RootState) => ({
+  ...INITIAL_STATE,
+  instanceId: state.instanceId
```

Comment:
What is `instanceId`?

Reply:
It's super fair point about name being ambiguous, I'll update and add a comment to the slice explaining it's purpose.

So because aggregation plugin is a "scoped plugin", meaning it's created for every instance of the collection tab, but the `Inflight` and `Debounce` maps that I added are singletons in the scope of the redux slice module I had to add some way of differentiating  between runs done on different tabs, otherwise I would be canceling and debouncing `executeAggregation` calls for the same step in every tab, which can cause problems (currently it is possible to open a tab, start a long running aggregation, switch to another tab and continue to use it).

In a way it's not that great that aggregation plugin creates it's own id, it probably should be passed from the collection plugin (as this is the root for all the scoped plugins), but this would be a bigger change and due to how plugins work I would need to pass it over through all these `createContext` methods to eventually copy it over to the local state to access anyway, so I went for this instead.

Another option is to create these singleton maps per plugin "instance", I decided not to because then I'd need to store these maps in state anyway, this requires more work and similarly doesn't feel great as a solution either, so I opted to have just an identifier per instance.